cmd/all-in-one/setupcontext/setupcontext.go

```diff
@@ -0,0 +1,20 @@
+package setupcontext
+
+import "testing"
+
+var isAllInOne bool
+
+// SetAllInOne sets the internal flag to all in one on.
+func SetAllInOne() {
+	isAllInOne = true
+}
+
+// UnsetAllInOne unsets the internal all-in-one flag. Used in tests.
+func UnsetAllInOne(t *testing.T) {
```

Comment:
please remove the argument, it is not used and will increase the size of the binary.